src/set_theory/ordinal/notation.lean

```diff
@@ -791,6 +791,189 @@ begin
       exact (repr_opow_aux₂ _ ad a00 al _ _).2 } }
 end
 
+/-- Given an ordinal, returns `inl none` for `0`, `inl (some a)` for `a+1`, and
+  `inr f` for a limit ordinal `a`, where `f i` is a sequence converging to `a`. -/
+def fundamental_seq : onote → option onote ⊕ (ℕ → onote)
```

Comment:
I'm not sure if renaming this to match [ordinal.is_fundamental_sequence](https://leanprover-community.github.io/mathlib_docs/set_theory/cardinal/cofinality.html#ordinal.is_fundamental_sequence) would be a good idea.